### ISSUE TITLE
[Feature Request] "branches_to_be_notified" in gitlab_service_slack resource

### ISSUE THREAD

**Issue by awill1988** [gitlabhq/terraform-provider-gitlab#233]
Please consider adding the `branches_to_be_notified` argument from the Gitlab API

```
Branches to send notifications for. Valid options are “all”, “default”, “protected”, and “default_and_protected”
```

Reference: https://docs.gitlab.com/ee/api/services.html#createedit-slack-service

**Comment by RyPeck** [comment]
Appears that this work was completed in #242 but a new release has not yet been cut.

**Comment by armsnyder** [comment]
Closed via #242 and it has been released.
